apps/razers3/paramChooser.h

```diff
@@ -102,7 +102,8 @@ struct ParamChooserOptions
         // global input parameters
         totalN = 32;                // sequence length
         totalK = 2;                 // errors
-        optionLossRate = (TFloat)0.01;      // in
+//         optionLossRate = (TFloat)0.01;      // in
```

Comment:
this comment is intentional, because the original is the actually intended and documented behaviour. Once #1797 is addressed, this can be reverted.

I will merge this now, as is. If you disagree, I can make a [NOP] PR tomorrow,